weldx/asdf/util.py

```diff
@@ -18,6 +21,21 @@
     "notebook_fileprinter",
 ]
 
+
+def get_schema_path(schema: str) -> Path:
+    """Get the path to a weldx schema file."""
+    schema = schema.split(".yaml")[0]
+
+    p = WELDX_PATH / "asdf" / "schemas"
+    print(p)
+    schemas = list(p.glob(f"**/{schema}.yaml"))
```

Comment:
why globbing here, you could just do (p + schema).exists()?

Reply:
idea is to only require `single_pass_weld-1.0.0.schema` isntead of `weldx/datamodels/single_pass_weld-1.0.0` or other longer path strings

Reply:
I see. Thanks!